scripts/superdesk-desks/desks.js

```diff
@@ -172,7 +172,7 @@
                 }
 
                 scope.$watch('filter', queryItems);
-                scope.$on('task:stage', function(event, data) {
+                scope.$on('task:stage', function($event, data) {
```

Comment:
Consistency with the other event listeners that are using $event as argument name, no particular reason.